conrad/__main__.py

```diff
@@ -1,16 +1,26 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import absolute_import
-
-
+from __version__ import __version__
+import urllib.request, json
 __all__ = ("main",)
 
 
 def main():
     from conrad.cli import cli
 
     cli()
-
+def get_current_version():
+    url = "https://pypi.org/pypi/conference-radar/json"
+    with urllib.request.urlopen(url) as response:
+        data = json.loads(response.read().decode())
+    return data["info"]["version"]
 
 if __name__ == "__main__":
+    latest_version = get_current_version()
+
+    if __version__ < latest_version:
```

Comment:
A tuple is being compared with a string here. You can instead use:

<pre>
>>> from setuptools.version import pkg_resources
>>> pkg_resources.parse_version("0.1.0") < pkg_resources.parse_version("0.1.1")
True<pre>